roles/semaphore_deployment/templates/deploy.j2

```diff
@@ -1,22 +1,36 @@
 #!/bin/bash
 
 set -e
-branch="$1"
+IFS=' ' read -ra git_args <<< "$1"
+branch=${git_args[0]}
+commit=${git_args[1]}
```

Comment:
We don't check the commit var for invalid characters.

Reply:
Updated